app/models/form.rb

```diff
@@ -38,9 +43,10 @@ def initialize(form_path, start_year = "", sections_in_form = [], type = "lettin
       @questions = pages.flat_map(&:questions)
       @start_date = Time.iso8601(form_definition["start_date"])
       @end_date = Time.iso8601(form_definition["end_date"])
+      @submission_deadline = Time.zone.local(2023, 6, 9)
       @unresolved_log_redirect_page_id = form_definition["unresolved_log_redirect_page_id"]
     end
-    @display_end_date = start_year == 2022 ? Time.zone.local(2023, 6, 9) : @end_date
+    @edit_end_date = @end_date
```

Comment:
what is @end_date? I thought we were removing that? Or is it meant to represetn the middle end date where we can no longer create logs? Since we have lots of different end dates I think they should all have a descriptive name.

Reply:
I've added an ARD as part of this, I think that might answer your question, but let me know if it doesn't!

Reply:
Ah, in terms of the descriptive name. I agree! I didn't update it here because end_date is used in a lot of validations etc. There is a separate ticket (CLDC-2459) for the new logs end date, so I thought it might be better to update the name there, to keep the PRs smaller

Reply:
I would disagree with this except that you've picked up all three of these tickets